person.rb

```diff
@@ -24,6 +24,10 @@ def correct_name
     @name
   end
 
+  def add_rental(book)
+    @rentals.push(book) unless @rentals.include?(book)
+  end
```

Comment:
- [ ] This method should be adding a rental and not a single book. Kindly rectify this issue